main_test.go

```diff
@@ -225,6 +225,19 @@ func TestSetupFlagsConfigWithLinterMap(t *testing.T) {
 	assert.Equal(t, "", linter.Pattern)
 }
 
+func TestUnknownLinter(t *testing.T) {
```

Comment:
`TestValidateLinters` ? Could also go into `linters_test.go` if the soruce is moved files.

Reply:
Originally had named it that, but I thought it was disingenuous.  I'll beef the test up a bit and rename it.